### PR TITLE
Farore's Wind Anywhere and No Ammo Drops options

### PR DESCRIPTION
**Pull request by CharadeNut**

### CODE REVIEW

File: code/src/entrance.c [hunk 1]
```diff
@@ -28,6 +28,14 @@ void Scene_Init(void) {
 
     gRestrictionFlags[72].flags3 = 0; // Allows farore's wind in GTG
     gRestrictionFlags[94].flags3 = 0; // Allows farore's wind in Ganon's Castle
+    
+    //Allow Farore's Wind anywhere if given by settings
+    if (gSettingsContext.faroresWindAnywhere){
+        for (int i = 0; i < 99; i++){
+			gRestrictionFlags[i].flags3 &= ~0x30;
```
Comment (lioncash) [inline code/src/entrance.c]: We use spaces instead of tabs for indentation

File: code/src/item00.c [hunk 2]
```diff
@@ -16,7 +16,30 @@
 
 void EnItem00_rInit(Actor* thisx, GlobalContext* globalCtx) {
     EnItem00* item = THIS;
-
+	s16 item00Type = item->actor.params & 0x00FF;
```
Comment (lioncash) [inline code/src/item00.c]: Ditto here, where applicable for relevant lines.

File: code/src/entrance.c [hunk 3]
```diff
@@ -28,6 +28,14 @@ void Scene_Init(void) {
...
+			gRestrictionFlags[i].flags3 &= ~0x03;
```
Comment (HylianFreddy) [inline code/src/entrance.c]: This line makes it possible to use every item anywhere, not only FW. Is that intentional?

File: source/logic.cpp [hunk 4]
```diff
@@ -430,6 +437,7 @@ namespace Logic {
     BlueFire     = HasBottle && BlueFireAccess;
     Fish         = HasBottle && FishAccess;
     Fairy        = HasBottle && FairyAccess;
+	AmmoDrops    = NoDrops.IsNot(NODROPS_NOAMMO) && NoDrops.IsNot(NODROPS_NOAMMOORHEALTH);
```
Comment (lioncash) [inline source/logic.cpp]: Seems like tabs have snuck in here as well

**Comment by gamestabled** [comment]
regardless of above code comments and questions (i know some discussion happened in discord as well) i think we should freeze this until after 2,0

**Pull request closed without merging**